### ISSUE TITLE
Simplify CMake preset files

### ISSUE THREAD

**Issue by aminya** [aminya/cpp_vcpkg_project#56]
After #20 we have several CMake preset files. This is populating the root of the project. This should be simplified and these files should be merged into a single one. There's no need for this modularity especially because it makes the project look complex. 

## Upvote & Fund

@aminya is using [Polar.sh](https://polar.sh/aminya) so you can upvote and help fund this issue. The funding is received once the issue is completed & confirmed by you.

Thank you in advance for helping prioritize & fund our backlog!

<br/>

<a href="https://polar.sh/aminya/cpp_vcpkg_project/issues/56">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/aminya/cpp_vcpkg_project/issues/56/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/aminya/cpp_vcpkg_project/issues/56/pledge.svg">
</picture>
</a>

**Comment by aminya** [comment]
@ClausKlein Could you tackle this?

**Comment by aminya** [comment]
I simplified this a bit in #57
